src/main/java/service/WriteToFile.java

```diff
@@ -0,0 +1,5 @@
+package service;
+
+public interface WriteToFile {
```

Comment:
```suggestion
public interface FileWriter {
```
don't use verbs in the names of interfaces and classes